src/routes/jira/atlassian-connect/jira-atlassian-connect-get.ts

```diff
@@ -226,6 +226,15 @@ const	modules = {
 			conditions: adminCondition,
 			key: "gh-addon-subscription-repos",
 			location: "none"
+		},
+		{
```

Comment:
The modules defined here are only for the frontend pages, not for any backend routes. So this node route does not need to be defined as a module here.